code_review/ab/runICAs.m

```diff
@@ -0,0 +1,114 @@
+function [] = runICAs(inpath, outpath, task, varargin)
+%% this script runs ICA on all filtered/cleaned/epoched EEG dmt data.

```

Comment:
a windows style carriage return new line (\r\n)!
Not a problem for maltab but will lead to hours of debugging when it shows up in a text file that e.g. AFNI might want to read.

```
echo -e 'a\n\rb' | while read x; do echo "*${x}*"; done
*a*
b*
```